web/pages/affiliates/index.md

```diff
@@ -0,0 +1,28 @@
+title: Plasma Packages
+hidetitle: True
+author: Erik T. Everson
+
+The PlasmaPy's charter includes promoting open-source software for all of 
+the plasma community, and not just `plasmapy`.  We feel the entire community 
```

Comment:
```suggestion
The PlasmaPy Project's charter includes promoting open-source software for all of 
the plasma community, and not just PlasmaPy.  We feel the entire community 
```
I tend to use "`plasmapy`" when talking specifically about code, and "PlasmaPy" as the proper noun.

Reply:
I was referring to the package in this case.  This page contains a list of software packages and, hence, we're promoting packages so I was referring to `plasmapy`.

Reply:
I was thinking we could follow how [Astropy](https://www.astropy.org/) does it...  They use "`astropy`" in snippets of code, like `import astropy`...and instead write out "Astropy" when talking about it as the name of a package.

Reply:
Idk, to me having PlasmaPy refer to the project and PlasmaPy/`plasmapy` refer to the package is confusing.  In my mind things are a lot clearer if PlasmaPy is the overarching project and `plasmapy` is the package.  However, your the PlasmaPy Godfather so I'll defer to you.